sky/serve/serve_state.py

```diff
@@ -245,6 +255,15 @@ def set_service_status(service_name: str, status: ServiceStatus) -> None:
     _DB.conn.commit()
 
 
+def set_service_version(service_name: str, version: int) -> None:
+    """Sets the service version."""
+    _DB.cursor.execute(
+        """\
+        UPDATE services SET
+        current_version=(?) WHERE name=(?)""", (version, service_name))
+    _DB.conn.commit()
+
```

Comment:
Can this be combined with `add_or_update_version` function? The version used for calling `add_or_update_version` should always be the latest?